resources/views/welcome.blade.php

```diff
@@ -1,273 +1,810 @@
 <!DOCTYPE html>
 <html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
     <head>
-        <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta charset="utf-8" />
```

Comment:
Can we figure out how to prevent the formatter from adding `/>` to single tags like this.